src/ussd_public.js

```diff
@@ -368,12 +370,23 @@ go.app = function() {
                     new Choice('9_11', $("Between 9-11am")),
                     new Choice('2_5', $("Between 2-5pm"))
                 ],
-                next: 'state_voice_confirm'
+                next: function(choice) {
+                    return go.utils_project
+                        .update_msg_format_time(
+                            self.im,
+                            'voice',
```

Comment:
I know, it bugs me too. Unfortunately I started off the registrations with 'sms' and 'voice', so I'd have to change it in the registration JS apps and the Backend, so wasn't sure it's worth it.